vllm/model_executor/layers/quantization/fp8.py

```diff
@@ -1098,6 +1098,27 @@ def apply(
                     expert_map=expert_map,
                     apply_router_weight_on_input=apply_router_weight_on_input,
                 )
+        # self.fused_experts is set in init_prepare_finalize,
+        # do not delete this branch
```

Comment:
I think no need to say "do not delete"
```suggestion
        # self.fused_experts is set in init_prepare_finalize
```

Reply:
Fixed